### PR TITLE
Enable log forwarding from Azure SDK.

### PR DESCRIPTION
**Pull request by JoshLove-msft**
Addresses https://github.com/Azure/azure-sdk-for-net/issues/20085

### CODE REVIEW

File: sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusSessionsEndToEndTests.cs [hunk 1]
```diff
@@ -264,31 +275,40 @@ public async Task ServiceBusSessionSub_SessionLocks()
...
+
                 await jobHost.StopAsync();
             }
         }
 
+        private static List<LogMessage> GetLogMessages(IHost host)
+        {
+            List<LogMessage> logMessages = host.GetTestLoggerProvider().GetAllLogMessages()
+                // Filter out Azure SDK and custom processor logs for easier validation. Intentionally do the error check after as the
+                // Service Bus SDK currently logs errors when stopping the processor.
```
Comment (pakrym) [inline sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusSessionsEndToEndTests.cs:306]: Do we have a work item to fix this?

**Review by pakrym — APPROVED** [review]

File: sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusSessionsEndToEndTests.cs (hunk 1 above)
Comment (JoshLove-msft) [reply sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusSessionsEndToEndTests.cs:306]: Yes - https://github.com/Azure/azure-sdk-for-net/issues/19098#issuecomment-813126915

Comment (pakrym) [reply sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusSessionsEndToEndTests.cs:306]: Link it from the comment please.

**Review by jsquire — APPROVED** [review]

**Pull request merged**